src/operator/cudnn_activation-inl.h

```diff
@@ -57,8 +57,19 @@ class CuDNNActivationOp : public Operator {
       data = in_data[activation::kData].get_with_shape<gpu, 4, real_t>(dshape, s);
       out = out_data[activation::kOut].get_with_shape<gpu, 4, real_t>(dshape, s);
     } else {
-      data = in_data[activation::kData].get<gpu, 4, real_t>(s);
-      out = out_data[activation::kOut].get<gpu, 4, real_t>(s);
+      Shape<4> dshape;
+      index_t size_left = in_data[activation::kData].Size();
```

Comment:
these are bug fixes for cudnn_activation. It used to only accept 2 or 4 dimensional input.